pkgs/tools/video/yamdi/default.nix

```diff
@@ -0,0 +1,36 @@
+{
+  stdenv,
+  fetchurl,
+  ...
+}:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  name = "yamdi-${version}";
+  version = "1.9";
+
+  # Source repo is also available here:
+  # https://github.com/ioppermann/yamdi
+  src = fetchurl {
+    url = "https://downloads.sourceforge.net/project/yamdi/yamdi/${version}/yamdi-${version}.tar.gz";
+    sha256 = "4a6630f27f6c22bcd95982bf3357747d19f40bd98297a569e9c77468b756f715";
+  };
+
+  buildFlags = "CC=${stdenv.cc}/bin/cc";
+
+  installPhase = ''
+    mkdir -p $out/bin
+    install -m 0755 yamdi $out/bin/yamdi
+
+    mkdir -p $out/share/man/man1
+    install {,$out/share/man/}man1/yamdi.1
+  '';
+
+  meta = {
+    description = "Yet Another MetaData Injector for FLV";
+    homepage = "http://yamdi.sourceforge.net/";
+    license = licenses.bsd3;
+    platforms = platforms.all;
```

Comment:
Don't you want to maintain this package? You can become a maintainer by adding your github handle + name/email to `lib/maintainers.nix`.